.spelling

```diff
@@ -64,6 +64,7 @@ hostname
 hostnames
 metaschema
 metadata
+multitenancy
```

Comment:
I'm going to suggest that we use "multi-tenancy" and "multi-tenant" with hyphens. This is consistent with Jeremy's 2023 CUG paper (and the references) on the topic.